Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/SurveySettings.kt

```diff
@@ -1,7 +1,41 @@
 package de.rki.coronawarnapp.datadonation.survey
 
+import android.content.Context
+import com.google.gson.Gson
+import de.rki.coronawarnapp.datadonation.OneTimePassword
+import de.rki.coronawarnapp.util.di.AppContext
+import de.rki.coronawarnapp.util.preferences.clearAndNotify
+import de.rki.coronawarnapp.util.serialization.BaseGson
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class SurveySettings @Inject constructor()
+class SurveySettings @Inject constructor(
+    @AppContext val context: Context,
+    @BaseGson val gson: Gson
+) {
+
+    companion object {
+        private const val KEY_OTP = "one_time_password"
```

Comment:
minor:
Constants in Kotlin dont need to be in companion object and preferred to be placed top level